SageMaker/02_classification.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<p>I would probably move this code after the endpoint example, since you don't use the <code>Trainer</code> immediately. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>